service/src/main/java/bio/terra/landingzone/library/landingzones/management/LandingZoneBaseResourceType.java

```diff
@@ -0,0 +1,22 @@
+package bio.terra.landingzone.library.landingzones.management;
+
+import com.azure.core.util.ExpandableStringEnum;
+import java.util.Collection;
+
+public final class LandingZoneBaseResourceType
+    extends ExpandableStringEnum<LandingZoneBaseResourceType> {
+  public static final LandingZoneBaseResourceType AZURE_VNET =
```

Comment:
Checking my understanding: VNet is a "base" resource because other resources in the landing zone (e.g. AKS, Batch account) depend on it -- right?

Whereas other LZ resources have no cross dependencies so deletion order is unimportant.

This works for now, just wondering if we'll need to model dependencies differently in the future (e.g. maybe we'll need a 3-phase deletion or something).

Reply:
Correct. And I agree, we may need to revisit this in the future if required.